crates/uv-cli/src/lib.rs

```diff
@@ -4672,6 +4672,19 @@ pub struct PythonPinArgs {
     /// `requires-python` constraint.
     #[arg(long, alias = "no-workspace")]
     pub no_project: bool,
+
+    /// Pin to a specific Python version.
+    ///
+    /// Writes the pinned Python version to a .python-version file in the uv user configuration
```

Comment:
I think we tend to wrap this file name in backticks to

```suggestion
    /// Writes the pinned Python version to a `.python-version` file in the uv user configuration
```